### DISCUSSION TITLE
How to all taxonomies add one field in one go

### DISCUSSION THREAD

**Discussion by Raza1631** [statamic/cms#8597] [Q&A]
Sometimes, you may need to update multiple taxonomies in your Statamic CMS by adding a new field.

**Comment by duncanmcclean** [comment]
Am I correct in understanding that you want to add a new field to a taxonomy, then set it to a certain value on all terms inside that taxonomy?

If so, there's nothing within the Control Panel that'll let you do that. I think you'll need to write a script to [query all of the terms](https://statamic.dev/repositories/term-repository#querying) in that taxonomy, then set the value & update the terms.
